Healthy/Classes/Modules/Splash/SplashViewController.swift

```diff
@@ -36,11 +36,10 @@ final class SplashViewController: UIViewController {
 
 // MARK: - Actions
 
-private extension SplashViewController {
+ extension SplashViewController {
     @IBAction func didTapStartCooking(_ sender: Any) {
-        let viewModel = LoginViewModel()
```

Comment:
We need the navigation logic. Currently, you can use both until we remove the navigation logic.

Reply:
I forgot to remove it after I had deleted the navigation logic

Reply:
Do we still need to put the navigation logic? I think the navigation was fixed in SceneDelegate by Mohamed recently